baselines/train/configs.py

```diff
@@ -42,6 +42,7 @@
     'commons_harvest__private_property_1',
     'commons_harvest__open_disable_zapping_0',
     'commons_harvest__open_disable_zapping_1',
+    'commons_harvest__farmer_0',
```

Comment:
We are putting at least two eval scenarios per substrate
```suggestion
    'commons_harvest__farmer_0',
     'commons_harvest__farmer_1',
```